netmaster/main.go

```diff
@@ -153,7 +153,7 @@ func execOpts(opts *cliOpts) {
 	}
 	log.Infof("Control IP:Port %s:%s", controlIP, controlURL[1])
 
-	if opts.clusterMode == "docker" {
+	if opts.clusterMode == "docker" || opts.clusterMode == "docker-compose" {
```

Comment:
clusterMode is the mode in netmaster (docker vs kubernetes vs mesos). We need to know that we are in a special case of docker so that we avoid implicitly creating the docker network. Please suggest a name if you want to change it.